Automattic/newspack-plugin
# fix(ras-acc): various transactional email fixes

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207817176293825/1207990881852220/f

This PR addresses several style issues pointed out in the asana task above:

1. OTP Font and letter spacing:
![Screenshot 2024-08-13 at 16 09 03](https://github.com/user-attachments/assets/33bdb946-b9d4-4b4d-a48b-5ea2d03e46af)

2. Headings weight and size:
Email heading should be 46px and 500 font weight:
![Screenshot 2024-08-13 at 16 11 33](https://github.com/user-attachments/assets/fc6f149d-1111-41c3-9602-3d13255a0e93)

Frequently asked questions section should be 28px and 500 font weight:
![Screenshot 2024-08-13 at 16 12 00](https://github.com/user-attachments/assets/13aaf70b-bacb-46db-bf0d-8b1c121ea6a5)

3. Conditional rendering of footer site name/address hyphen and comma:
![Screenshot 2024-08-13 at 16 15 08](https://github.com/user-attachments/assets/53fc462c-a77a-47db-92f2-306298649bf9)


### How to test the changes in this Pull Request:

1. Reset all email templates via the reset buttons in Newspack > Engagement > Show Advanced Settings > Transactional Emails and Newspack > Reader Revenue > Emails menus
2. View each of the emails in the editor and confirm the above three points:
  - OTP font, font size, and letter spacing
  - Headings font size and weight
  - Footer site name/address:
      - Remove or add a store address via Woocommerce > Settings, reset emails, and verify site name/address section either appears with a hyphen and comma, or as just the site name 
3. Send test emails for each template and verify the above three points for the actual emails 

### Other information:
